src/runtime/virtcontainers/agent.go

```diff
@@ -190,4 +191,6 @@ type agent interface {
 
 	// getAgentMetrics get metrics of agent and guest through agent
 	getAgentMetrics(context.Context, *grpc.GetMetricsRequest) (*grpc.Metrics, error)
+
+	task.ImageService
```

Comment:
I would prefer to introducing kata's own image API between runtime and agent instead of relying on containerd's definition. Even if they end up the same, it would allow us to maintain the agent API internally, since it is internal API afterall.